sources/install/package_cloud.sh

```diff
@@ -51,9 +51,18 @@ function install_scout() {
     add-to-list "scout,TODO,TODO"
 }
 
+function install_azurecli(){
+  colorecho "Installing Azure cli"
+  fapt azure-cli
+  add-history azurecli 
+  add-test-command "az version"
+  add-to-list "azurecli,https://learn.microsoft.com/en-us/cli/azure/,Command-line interface for Azure."
```

Comment:
Apt tools must be installed separately.

You have to do a install_cloud_apt_tools() function that you call from the package_cloud function.

Example : https://github.com/ThePorgs/Exegol-images/blob/dev/sources/install/package_misc.sh#L6